index.rst

```diff
@@ -190,7 +190,7 @@ Key Resources
 * Source code
     * `Browse online <https://github.com/python/cpython/>`_
     * `Snapshot of the *main* branch <https://github.com/python/cpython/archive/main.zip>`_
-    * `Daily OS X installer <https://buildbot.python.org/daily-dmg/>`_
+    * `Daily macOS installer <https://buildbot.python.org/daily-dmg/>`_
```

Comment:
We should just delete this. That buildbot hasn't existed for years.

Reply:
Thanks, deleted!